modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/EditObjectField.tsx

```diff
@@ -144,8 +139,7 @@ export default function EditObjectField({
 			readOnly={readOnly}
 			title={Liferay.Language.get('field')}
 		>
-			{(Liferay.FeatureFlags['LPS-170122'] && isDefaultStorageType) ||
-			values.businessType === 'Picklist' ? (
```

Comment:
The way that this works in the master, a Picklist field has the 'advanced' tab, even if the storage type is _nonDefault_.
The way that you perform here, a Picklist field has no the 'advanced', if the storage type is _nonDefault_.
It is your intention to change this behavior?
I mean, is expected to hide the **Default Value** tab, when the objectDefinition has _nonDefault_ storage type?

Reply:
I will request a review for alpha squad.

Reply:
@murilostodolni If you make the condition like `isDefaultStorageType || values.businessType === 'Picklist'` the readOnly will not be applied only for Picklists. We cannot not show the advanced tab for Picklists because if the user marks it as a state they will not be able to save the field until they define the default value, that appears in the advanced tab.